scripts/lib/CIME/SystemTests/system_tests_compare_two.py

```diff
@@ -21,6 +21,17 @@
     that's needed in both cases. This is called before _case_one_setup or
     _case_two_setup.
 
+(2) _case_one_custom_prerun_action(self):
```

Comment:
Above, change "MAY require the following method" to "MAY require the following method**s**" (i.e., make 'method' plural)